Python/Product/EnvironmentsList/PipExtensionProvider.cs

```diff
@@ -143,6 +143,11 @@ public sealed class PipExtensionProvider : IEnvironmentViewExtension {
         internal async Task<IList<PipPackageView>> GetInstalledPackagesAsync() {
             string[] args;
 
+            if (!CanExecute) {
+                // Invalid configuration, so assume no packages

```

Comment:
They didn't enter the path to the environment. Everything is disabled and there are big blank spots next to some icons.

I don't disagree we should help them do the right thing, but this isn't the place to add that. Probably the only option that will work reliably here is to block them at the Create step until they've gotten everything right.